docs/bugs_and_glitches.md

```diff
@@ -2629,3 +2630,18 @@ If `IsInArray` returns `nc`, data at `bc` will be executed as code.
  	jr nz, .loop
  	ret
 ```
+
+
+### Options Menu fails to clear joypad state on initialization
+
+This bug (or feature!) results in all options being shifted left or right if the respective direction is pressed on the same frame the options menu is opened. The bug also exists in pokeyellow and pokegold.
+
+**Fix:** Edit `_Option` in [engine/menus/options_menu.asm](https://github.com/pret/pokecrystal/blob/master/engine/menus/options_menu.asm):
+
+```diff
+ _Option:
++	call ClearJoypad
+	ld hl, hInMenu
+	ld a, [hl]
+	push af
```

Comment:
```suggestion
  _Option:
+	call ClearJoypad
 	ld hl, hInMenu
 	ld a, [hl]
 	push af
```

Reply:
Gee Rangi, how come your mom lets you have *TWO* spaces in front of `_Option:`?

Reply:
```suggestion
 _Option:
+	call ClearJoypad
 	ld hl, hInMenu
 	ld a, [hl]
 	push af
```

Reply:
```suggestion
 _Option:
 -; BUG: Options menu fails to clear joypad state on initialization (see docs/bugs_and_glitches.md)
+	call ClearJoypad
 	ld hl, hInMenu
 	ld a, [hl]
 	push af
```
Likewise that `; BUG:` comment should be `added` to the same location in `engine/menus/options_menu.asm`